pybamm/models/submodels/electrolyte_diffusion/composite_diffusion.py

```diff
@@ -95,14 +95,3 @@ def set_initial_conditions(self, variables):
         c_e = variables["Electrolyte concentration"]
 
         self.initial_conditions = {c_e: self.param.c_e_init}
-
-    def set_boundary_conditions(self, variables):
```

Comment:
I think at some point we moved these into the submodels to be clearer what the submodels are. It's not too much repeated code.

Reply:
Makes sense, I will put them back. I did it this way as you also need to provide boundary conditions for the constant concentration model so that `grad(c_e)` ends up having the correct shape.